tests/unit/responses/world_scientific/wsp_record.xml

```diff
@@ -34,7 +34,6 @@
                 </contrib>
             </contrib-group>
             <aff id="S0219025717500060AF001"><label><sup>1</sup></label>Department, University, City, City_code 123456, <country>C. R. Country_2</country></aff>
-            <aff id="S0219025717500060EM001"><email>name_1@domain.com</email></aff>
```

Comment:
it looks like this is the way WSP do emails after all. Can you change the `JatsParser.emails` property to take also this case into account?